CHANGELOG.md

```diff
@@ -59,6 +59,7 @@
 - Minor: Added `/streamlink` command. Usage: `/streamlink <channel>`. You can also use the command without arguments in any twitch channel to open it in streamlink. (#2443, #2495)
 - Minor: Humanized all numbers visible to end-users. (#2488)
 - Minor: Added a context menu to avatar in usercard. It opens on right-clicking the avatar in usercard. (#2517)
+- Minor: Added "/popout" command - opens current splits web chat if no arguments are provided, if arguments are provided it will open provided channel. Usage /popout <channel> (#2556)
```

Comment:
```suggestion
- Minor: Added "/popout" command - opens current split's web chat if no arguments are provided, if arguments are provided it will open provided channel. Usage: /popout <channel> (#2556)
```
Channeling my inner @Felanbird